src/styles/pages/_live.scss

```diff
@@ -0,0 +1,137 @@
+@import '../settings/colors';
+@import '../tools/breakpoints';
+
+.w-live-landing-page {
+  color: $GREY_800;
+
+  h1,
+  h2 {
+    color: $GREY_900;
+  }
+}
+
+.w-live-top {
+  // We need this as the live background extends outside the normal l/r of the
+  // page. Contain it so it doesn't let users scroll.
+  overflow-x: hidden;
+}
+
+.w-event-section {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  margin: 64px auto;
+  max-width: 960px;
+  text-align: center;  // for mobile
+
+  .w-event-heading {
```

Comment:
I'm not sure if it's worth it to nest in this way. It's increasing selector specificity and bytes over the wire without producing a benefit since .w-event-heading already scopes the styles pretty well on its own.